arch/arm/src/lpc17xx_40xx/lpc17_40_can.c

```diff
@@ -72,6 +91,14 @@
 
 #if defined(CONFIG_LPC17_40_CAN1) || defined(CONFIG_LPC17_40_CAN2)
 
+#if defined(CHRDEV_CAN) && defined(SOCKET_CAN)
+# error "Both chrdev CAN or SocketCAN have been enabled"
+#endif
+
+#if defined(CHRDEV_CAN)
+#define lpc17_40_can_s lpc17_40_can_s
```

Comment:
Should it be: "#define lpc17_40_s can_dev_s" ?

Reply:
Oh somehow missed that thank your for pointing it out, I've changed it and tested again chrdev mode as well.